docs/components/Date.md

```diff
@@ -0,0 +1,87 @@
+
+The [GDS guidance for dates](https://design-system.service.gov.uk/patterns/dates/) was used as a basis for the component and [WCAG2.2](https://www.w3.org/WAI/WCAG22/Understanding/) compliance is offered out of the box.
+
+## Inputs and Outputs
+
+The component has various custom inputs which dictate the functionality, styling and error handling of the component.
+
+We show the display name of the property as well as its advanced name in the form **Display name - AdvancedName**.
+
+There are also custom output properties which can be used to reference component input values and state in other controls.
+
+### Inputs
+
+- **Title - Title** - Text to be shown as the title.
+    - **Default:** What is your date?
+- **Subtitle - Subtitle** - Text to be shown as the subtitle.
+    - **Default:** Enter a valid date
+- **Show title - ShowTitle** - Boolean value that controls the title visibility.
+    - **Default:** True
+- **Show subtitle - ShowSubtitle** - Boolean value that controls the subtitle visibility.
+    - **Default:** True
+- **Required - Required** - Boolean value that controls whether the date input is required or optional.
+    - **Default:** True
+- **Submit enabled - SubmitEnabled** - Enables submit functionality. When this is false, live validation is used. Need to hook up component to a button control to trigger validation with submit functionality.
+    - **Default:** True
+- **Submit - Submit** - Boolean to trigger the on submit functionality. Only works when SubmitEnabled is set to true.
+    - **Default:** False
+- **Theme - Theme** - Configures the styling of component elements.
+    - **Default:** See [Theme Documentation](../Theme.md).
+- **Show day - ShowDay** - Controls the visibility of the day field.
+    - **Default:** True
+- **Show year - ShowYear** - Controls the visibility of the year field.
+    - **Default:** True
+- **Default day - DefaultDay** - Day to be used in the date output value when the day field is not visible.
+    - **Default:** 01
+- **Default year - DefaultYear** - Year to be used in the date output value when the year field is not visible.
+    - **Default:** 1972
+- **Max date - MaxDate** - Maximum date that will be accepted as an input into the component.
+    - **Default:** DateValue("31/12/9999")
+        - This is the limit of the maximum date input property, so any date set should not exceed this default.
+- **Min date - MinDate** - Minimum date that will be accepted as an input into the component.
+    - **Default:** DateValue("01/01/1900")
+        - This is the limit of the minimum date input property, so any date set should not precede this default.
+
+### Outputs
+
+- **Value - Value** - Outputs the sanitised date input.
+- **Date format - DateFormat** - Outputs common date formats, based on the fields visible in the component.
```

Comment:
I think this would be clearer if it said something like `Outputs the format pattern of the date, based on the fields visible in the component` ?
This makes it sound to me like it could output multiple formats?